hazelcast/cluster.py

```diff
@@ -214,15 +214,15 @@ def _apply_new_state_and_fire_events(self, current, snapshot):
                     try:
                         handler(removed_member)
                     except:
-                        self.logger.exception("Exception in membership lister", extra=self._logger_extras)
+                        _logger.exception("Exception in membership lister")
```

Comment:
A stupid question: don't we need to log some information about the raised exception here and on L224?

Reply:
`.exception()` does that automatically. It inspects the current exception stack (and finds the one we caught) and logs it